store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/FederatedOperationChainValidator.java

```diff
@@ -101,10 +100,7 @@ protected void validateViews(final Operation op, final User user, final Store st
                 if (graphIdValid) {
                     currentResult = new ValidationResult();
                     clonedOp.graphIdsCSV(graphId);
-                    // Deprecated function still in use due to Federated GetTraits bug with DYNAMIC_SCHEMA
-                    if (!graphSerialisable.getGraph().getStoreTraits().contains(StoreTrait.DYNAMIC_SCHEMA)) {
```

Comment:
This only used to run on proxy stores and federated stores but now on all, is this okay?

Reply:
"**Not** (Traits contains Dynamic Schema)"